test/unit/client-href-builder.js

```diff
@@ -59,5 +59,16 @@ describe('client : href-builder :', () => {
             .to.equal('http://localhost:3030/hello-world/1.0.0/?p1=v1&p2=v%202');
       });
     });
+
+    describe('when there are parameters in both "options" and "component"', () => {
+      it('it should return a valid request for the component with the parameters set as URL query params', () => {
+        let options = { parameters: { p1: 'v1', p2: 'v 2' } };
+        let component = { name: 'hello-world', version: '1.0.0', parameters: { message: 'hello' } };

```

Comment:
I would ensure we include a common parameter to ensure our merge goes in the right way, such as { message: 'hello', p1: 'v3'} and be sure qs has p1=v1